CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ All notable changes to this project will be documented in this file.
    - `removeAll(_ elements: [Element])` (new overload) for `Equatable` elements in [[PR]](https://github.com/goktugyil/EZSwiftExtensions/pull/416) by *dfrib*
    - `removeAll(_ elements: [Element])` (new overload) for `Hashable` elements in [[PR]](https://github.com/goktugyil/EZSwiftExtensions/pull/416) by *dfrib*
 
+2. **Double**
+   - `var toCGFloat` in PR by *Vic-L*
```

Comment:
I think you need to put the PR number here. You can look at some of the changelog entries to understand how its done.